tensorflow/lite/micro/kernels/arc_mli/conv_slicing_test.cc

```diff
@@ -0,0 +1,509 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Can you explain why this separate test file is needed, and how it relates to the standard tests? In general we expect sub-classed implementations of ops to pass the standard tests in the folder above, so it would be good to understand why an implementation-specific test is needed, and if it replaces the standard one?

Reply:
The purpose of these tests is to check that slicing logic doesn’t affect result of operations.

These tests don’t replace standard ops tests, only complement the all test set. And only in case MLI for ARC platform is used during generation (which is handled in [arc_mli.inc]( https://github.com/tensorflow/tensorflow/pull/39077/files#diff-2f6c08e7e363da5eedc15ce82ebc44cfR85-R88) makefile). So such tests won’t be generated for other platforms.